packages/babel-helper-create-class-features-plugin/src/misc.js

```diff
@@ -98,10 +101,14 @@ export function extractComputedKeys(ref, path, computedPaths, file) {
       const ident = path.scope.generateUidIdentifierBasedOnNode(
         computedNode.key,
       );
+      nearestBlock.unshiftContainer(
+        "body",
+        t.variableDeclaration("let", [t.variableDeclarator(ident, null)]),
+      );
```

Comment:
Can we use `path.scope.push` instead? So we don't have to re-implement the logic to decide where to put the var.

Reply:
changed